CMakeLists.txt

```diff
@@ -3,11 +3,17 @@ project(png2tile)
 
 find_package(PNG REQUIRED)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
Just wondering why this change was required. Does it introduce some gnu only functions?

Reply:
Yes lets revert back to c++11